inst/tinytest/testPlot.R

```diff
@@ -937,6 +945,9 @@ expect_identical_xl(
   c("feature_0006", "feature_0002")
 )
 
+# Plotly Plots -----------------------------------------------------------------
+
```

Comment:
We need some tests where the call to `plotStudy()` succeeds. For example, you could run `plotPlotly` and then confirm that it returns JSON. You can confirm it has class "json" with `inherits()`. For example, the following checks if the object `x` inherits from class "json": `inherits(x, "json")`